charts/enterprise-operator/values-openshift.yaml

```diff
@@ -27,6 +27,7 @@ operator:
 # Environment variables prefixed with RELATED_IMAGE_ are used by operator-sdk to generate relatedImages section
 # with sha256 digests pinning for the certified operator bundle with disconnected environment feature enabled.
 # https://docs.openshift.com/container-platform/4.14/operators/operator_sdk/osdk-generating-csvs.html#olm-enabling-operator-for-restricted-network_osdk-generating-csvs
+  version: 1.29.0
```

Comment:
Why do we have this new line ?

Reply:
For some reason the original PCT commit has it: https://github.com/10gen/ops-manager-kubernetes/pull/3911/commits/b54f2f8b3229f811ac0c8cf8caf40b2da35ba1a0#diff-d6dd1644947c68b9a6ad9a498ae12f73802d316aeef41f57d1de0fd9294710b2R30.